docs/docs/developers/guides/js_apps/pay_fees.md

```diff
@@ -140,7 +152,7 @@ See this [section](../../reference/environment_reference/cli_wallet_reference.md
 This method of fee payment will only work for environments where a sponsored fee paying contract is deployed.
 The sandbox comes with a sponsored fee paying contract deployed, so this can be used to pay for transactions without needing to bridge fee juice.
 To use sponsored FPCs in other environments, they will need to be deployed and funded with fee juice.
-Creating the SponsoredFPC is as simple as importing it and passing it the PXE:
+Using the SponsoredFPC payment method is as simple as importing it, registering it and passing it the PXE:
```

Comment:
Previous wording didn't assume an enshrined FPC, but since it's now enshrined as "the" SponsoredFPC (`FPCContract`).
The change might just be the previous line, but can revise this in the following branch.

Reply:
im updating the wording here to not assumed enshrined Sponosored FPC as well. update "the" to "a"